contracts/ethereum/src/PaymentRegistry.sol

```diff
@@ -39,18 +41,18 @@ contract PaymentRegistry is Initializable, OwnableUpgradeable, UUPSUpgradeable {
     // no constructors can be used in upgradeable contracts. 
     function initialize(
         address snMessaging,
-        uint256 StarknetEscrowAddress_,
         uint256 StarknetEscrowClaimPaymentSelector_,
         address marketMaker_,
-        address ZKSyncDiamondProxyAddress) public initializer { 
+        address ZKSyncDiamondProxyAddress,
+        bytes4 ZKSyncEscrowClaimPaymentSelector_) public initializer { 
         __Ownable_init(msg.sender);
         __UUPSUpgradeable_init();
 
         _snMessaging = IStarknetMessaging(snMessaging);
         _ZKSyncDiamondProxy = IZkSync(ZKSyncDiamondProxyAddress);
 
-        StarknetEscrowAddress = StarknetEscrowAddress_;
```

Comment:
Why remove StarknetEscrowAddress from init function

Reply:
This value was initialized with "0" because when deploying PaymentRegistry, Escrow was not deployed yet.
Now this value is only set with the setStarknetEscrowAddress() called after the escrow is deployed.
(Note the setting of this function was always this way, the only thing changed is removing the initial set var=0)